MapboxGeocoder/MBPlacemark.swift

```diff
@@ -486,6 +491,13 @@ open class GeocodedPlacemark: Placemark {
         return properties?.maki
     }
     
+    /**
+     A numerical score from 0 (least relevant) to 0.99 (most relevant) measuring
+     how well each returned feature matches the query. You can use the this
+     property to remove results that don’t fully match the query.
+     */
+    open var relevance: Double?
```

Comment:
Float would probably have sufficient precision, but I guess there’s no harm in using double here.